src/components/BookContainer.js

```diff
@@ -0,0 +1,29 @@
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import Book from './Book';
+import AddBookButton from './AddBookButton';
+import { fetchBooks } from '../redux/books/booksSlice';
+
+/* eslint-disable */
+const BookContainer = ({ app_id }) => {
+  const books = useSelector((state) => state.books.books);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(fetchBooks(app_id));
+  }, [dispatch, app_id]);
+
+  return (
+    <div className="container">
+      {books.map((book) => (
+        <div key={book.item_id}>
+          <Book book={book} />
+        </div>
+      ))}
+      <div className="part-devider"></div>
+      <AddBookButton app_id={app_id} />
+    </div>
```

Comment:
Great work by creating this component 👍🏼  ,  one very minor styling adjustement needed is to reduce the width of this container or increase its margin 

**figma design**
![Capture d’écran du 2023-08-04 22-19-52](https://github.com/akbarsazish/Bookstore-React/assets/59707859/e048148f-1c82-45eb-a693-8e53b3d6ea38)

**your design**
![Capture d’écran du 2023-08-04 22-19-27](https://github.com/akbarsazish/Bookstore-React/assets/59707859/030b6468-7430-4911-9217-1c0e3ecbf6e1)